django_opentracing/tracer.py

```diff
@@ -63,12 +65,12 @@ def _apply_tracing(self, request, view_func, attributes):
         span = None
         operation_name = view_func.__name__
         try:
-            span_ctx = self._tracer.extract(opentracing.Format.HTTP_HEADERS, headers)
-            span = self._tracer.start_span(operation_name=operation_name, child_of=span_ctx)
+            span_ctx = opentracing.tracer.extract(opentracing.Format.HTTP_HEADERS, headers)
```

Comment:
This surprises me... usually the pattern is to *default* to the `opentracing.tracer` but to allow for overrides (a la the `tracer` param to `__init__`). That way it remains possible to use the django opentracing instrumentation without *requiring* the use of the global tracer.

Reply:
That's a fair point. My rationale here was to allow for lazy initialization, which requires a global object somewhere..

I'll change this so you can use the instrumentation without the global tracer. (But obviously then the user needs to deal with lazy initialization themself)

Thanks for the input :)

Reply:
@bhs How's that?